lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -43,7 +45,7 @@
 from xmodule.modulestore.xml import XMLModuleStore
 from opaque_keys.edx.locations import SlashSeparatedCourseKey
 from instructor_task.models import InstructorTask
-
+from django_comment_client.management_utils import rename_user as rename_user_util
```

Comment:
You don't need to change this, but alternatively,  you could have done something like this to avoid the naming conflict:

```python
from django_comment_client import management_utils
management_utils.rename_user(old, new)
# or even
from django_comment_client import management_utils as utils
utils.rename_user(...)
```